TYPO3.Flow/Tests/Unit/Core/BootstrapTest.php

```diff
@@ -49,4 +49,14 @@ public function isCompileTimeCommandControllerChecksIfTheGivenCommandIdentifierR
 
         $this->assertSame($expectedResult, $bootstrap->isCompiletimeCommand($givenCommandIdentifier));
     }
+
+    /**
+     * @test
+     * @expectedException \TYPO3\Flow\Exception
+     */
+    public function resolveRequestHandlerThrowsUsefulExceptionIfNoRequestHandlerFound()
+    {
+        $bootstrap = $this->getAccessibleMock(Bootstrap::class, array('dummy'), array(), '', false);
+        $bootstrap->_call('resolveRequestHandler');
+    }
```

Comment:
Yes, this should be a test for this exact exception. Couldn't properly fix it up earlier as I'm on vacation here and only have short time spans before my little one wants attention.